packages/ra-tree-core/src/reducers/data.ts

```diff
@@ -0,0 +1,176 @@
+import { Reducer } from 'redux';
```

Comment:
You're duplicating most of the logic of the ra-core data reducer. Why can't you just store the extra data that you need in this reducer?

Reply:
I would have to dispatch a *fake* `CRUD_LIST_SUCCESS` for that but that could work